tests/mock_server.py

```diff
@@ -45,8 +46,12 @@ def _run(self):
         app.router.add_route('*', '/{anything:.*}', self.stream_handler)
 
         try:
-            aiohttp.web.run_app(app, host=host, port=self._port,
-                                loop=self._loop, handle_signals=False)
+            # We need to mock `.get_event_loop` function and return
+            # `self._loop` explicitly because from `aiohttp>=3.0.0` we can't
+            # pass `loop` as a kwargs into `run_app`.
+            with mock.patch('asyncio.get_event_loop', return_value=self._loop):
+                aiohttp.web.run_app(app, host=host, port=self._port,
```

Comment:
We can use new AppRunner stuff as result we do not need ugly patch
https://docs.aiohttp.org/en/stable/web_advanced.html?highlight=AppRunner#application-runners